rust-src/Cargo.lock

```diff
@@ -351,7 +351,7 @@ dependencies = [
 
 [[package]]
 name = "concordium-contracts-common"
-version = "5.1.1"
+version = "5.2.0"
```

Comment:
Other lock files in this repository also need to be updated.

Reply:
Great, I'll do that 😊